src/apps/chatter/hooks/use-room.js

```diff
@@ -84,18 +99,20 @@ export function useRoom (roomId) {
     // console.log('SENDING NEW MESSAGE WITH TOKEN: ', token)
     sendMessage({
       command: NEW_ENTRY,
+      password,
       text,
       token
     })
   }
 
-  const sendMessage = ({ command, text, token }) => {
+  const sendMessage = (message) => {
+    console.log('SENDING MESSAGE ', message)
     try {
-      socketRef.current.send(JSON.stringify({ command, text, token }))
+      socketRef.current.send(JSON.stringify(message))
     } catch (err) {
       console.log(err.message)
     }
   }
 
-  return [entries, sendNewEntry]
+  return [entries, sendNewEntry, isAuthorized, isInitialized]
```

Comment:
this should be an object because there are more than 2 refs being returned

Reply:
Done like dinner